web/src/components/atoms/Icon/Icons/clear.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think no need to add the SVG file.
The same icon exists in `antd` icons its name `ClearOutlined`

Reply:
Oh didn't see that! Thanks